librabbitmq/win32/threads.h

```diff
@@ -27,7 +27,7 @@
 #ifndef AMQP_THREAD_H
 #define AMQP_THREAD_H
 
-#ifndef WINVER
+#if defined(WINVER) || defined(__MINGW32__) || defined(__MINGW64__)
```

Comment:
Are you sure? Previously, this only set a default value of WINVER, if it was not defined. 

Now, it overrides WINVER if it was defined, but still leaves it undefined if it was not defined before.

Previously, it would have defined WINVER even on MinGW, so the description of the PR/commit does not fit.

Reply:
Good catch.  It needs to be `!defined(WINVER)`.  Additionally there will be redefinition errors if mingw defines `WINVER`.